src/pages/Payment/components/Dropdown/ListCheck/ListCheck.css.ts

```diff
@@ -0,0 +1,48 @@
+import { style } from "@vanilla-extract/css";
+import { vars } from "@/shared/styles/tokens.css.ts";
+
+export const listCheck = style({
+  display: "flex",
+  width: "37.5rem",
+  height: "5.6rem",
+  padding: "1.6rem 1.8rem",
+  justifyContent: "space-between",
+  alignItems: "center",
+  background: vars.color.white,
+});
+
+export const listCheckLeft = style({
+  display: "flex",
+  alignItems: "center",
+  gap: "0.8rem",
+});
+
+export const listCheckCheckIcon = style({
+  display: "flex",
+  width: "1.8rem",
+  height: "1.8rem",
+  padding: "0.3rem 0 0.3rem 0.1rem",
+  justifyContent: "center",
+  alignItems: "center",
+  flexShrink: 0,
+});
+
+export const listCheckLabel = style({
+  fontFamily: vars.font.family.pretendard,
+  fontSize: vars.font.b2_b_16.size,
+  fontWeight: vars.font.b2_b_16.weight as any,
+  lineHeight: vars.font.b2_b_16.lineHeight,
+  letterSpacing: vars.font.b2_b_16.letterSpacing,
+  color: vars.color.gray80,
```

Comment:
```suggestion
import { fontStyle } from '@/shared/styles/fontStyle'; 

...fontStyle('b2_b_16')
```

으로 작성하시면 전부 코드를 작성하지 않아도 됩ㄴ디ㅏ !!! 다른 폰트들도 변경 부탁드릴게요

Reply:
넵 감사합니다:)!!!!!!